library/swift_disk.py

```diff
@@ -69,8 +69,12 @@ def main():
     if os.path.exists(part_path) and os.path.ismount(mount_point):
         module.exit_json(changed=False)
 
-    # create partitions
     if dev_path is not None:
+        # create label
+        cmd = ['parted', '--script', dev_path, 'mklabel', 'msdos']
+        rc, out, err = module.run_command(cmd, check_rc=True)    
+
```

Comment:
flotsam bootstrap already creates partition. If we want to recreate one here for vagrant then type should be gpt to handle > 2TB